{"name": "test_change_to_invalid_username", "status": "passed", "attachments": [{"name": "invalid username error message", "source": "daddbccd-ab86-4057-b195-9301366110e8-attachment.png", "type": "image/png"}, {"name": "log", "source": "e3b9ff31-c770-48dc-bfb9-9d31dd399b7d-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "654af3c6-bd57-455c-a3e7-eb6efd93509b-attachment.txt", "type": "text/plain"}], "start": 1723157808839, "stop": 1723157844462, "uuid": "fbed427f-42d4-4bf3-b359-2095b918e2be", "historyId": "dc0a9cd52b3898118b858a19189c6cf4", "testCaseId": "dc0a9cd52b3898118b858a19189c6cf4", "fullName": "test.web.test_change_to_invalid_username.ProfileInvalidUsernameChange#test_change_to_invalid_username", "labels": [{"name": "story", "value": "Change to Invalid Username"}, {"name": "feature", "value": "Username invalid - test"}, {"name": "parentSuite", "value": "test.web"}, {"name": "suite", "value": "test_change_to_invalid_username"}, {"name": "subSuite", "value": "ProfileInvalidUsernameChange"}, {"name": "host", "value": "DESKTOP-OPJ94K2"}, {"name": "thread", "value": "25184-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.web.test_change_to_invalid_username"}]}